Composer/plugins/vacore/template/scripts/DeploymentTemplates/function-template-with-preexisting-rg.json

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
Is this the same file as is currently present in the "boilerplate" that is copied into every new project?  If so, we should remove it from this template.  If not, we should make sure that there is not a file naming collision, otherwise the boilerplate copy may overwrite it.

https://github.com/microsoft/BotFramework-Composer/tree/main/Composer/plugins/samples/assets/shared/scripts

Reply:
I have cleaned up the template a good bit, to note this is a version of the declarative VA template that is maintained and reviewed in a separate repo.

Reply:
@GeoffCoxMSFT @benbrown  I am not going to address all of the template comments in this PR. The reason being that this template is a placeholder for the time being and will change as a whole in the next iteration of the creation work.